src/lib/containers/dataaccess/AccessRequirementSearchBox.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import AsyncSelect from 'react-select/async'
+import { SynapseClient } from '../../utils'
+import { useSynapseContext } from '../../utils/SynapseContext'
+
+export type AccessRequirementSearchBoxProps = {
+  onChange: (accessRequirementId?: string | undefined) => void
```

Comment:
Expected the ability to set the initial AR ID as well (send ACT to a page that has this filter applied via the page token, it should be visible in the search box)

Reply:
Good catch